### ISSUE TITLE
Ledvance Smart+ Panel Light 30 x 120: Interview failed because can not get node descriptor

### ISSUE THREAD

**Issue by gadgetmies** [Koenkk/zigbee2mqtt#10026]
## Information about the device + link

[Ledvance SMART+ PANEL 30 x 120cm Tunable White](https://www.ledvance.com/consumer/products/smart-home/smart-home-products-with-zigbee-technology/smart-home-luminaires/indoor-luminaires/wall-and-ceiling-luminaire-with-zigbee-technology-c6414)

## Issue

Zigbee2mqtt version: 1.22.1

When I turn on the device and wait a while, it seems that the zigbee-herdsman tries to pair the device, but fails to get the node descriptor from the device:
```
  zigbee-herdsman:controller:device:log Interview - quirks check for 'undefined'-'undefined'-'Unknown' +23s
  zigbee-herdsman:controller:device:log Interview - quirks did not match +0ms
  zigbee-herdsman:controller:device:log Interview - node descriptor request failed for '0xf0d1b8000013cd20', attempt 6 +0ms
  zigbee-herdsman:controller:device:log Interview - quirks check for 'undefined'-'undefined'-'Unknown' +0ms
  zigbee-herdsman:controller:device:log Interview - quirks did not match +1ms
  zigbee-herdsman:controller:device:log Interview - failed for device '0xf0d1b8000013cd20' with error 'Error: Interview failed because can not get node descriptor ('0xf0d1b8000013cd20')
  zigbee-herdsman:controller:device:log     at Device.interviewInternal (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/controller/model/device.ts:478:19)
  zigbee-herdsman:controller:device:log     at Device.interview (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/controller/model/device.ts:369:13)
  zigbee-herdsman:controller:device:log     at Controller.onDeviceJoined (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/controller/controller.ts:544:17)' +2ms
```

See the longer log excerpt at: https://pastebin.com/ZyP0gkYs

As the pairing fails I do not get the information required to add the device as instructed at: https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#instructions

Any ideas how to debug the issue further? I was rather surprised that the 30x120cm panel does not work, but the 60x60cm panel seems to be supported. Might it be possible to somehow use the 60x60cm panel configurations to build support for the 30x120cm panel and if so, how could I do that?

**Comment by gadgetmies** [comment]
It is also possible that the issue is with factory resetting the device. I followed the "5x 5s on + 5s off" procedure, but did not get any confirmation from the panel that the reset worked (i.e. the panel did not make any extra blinks or such). When turning on the device after the last on/off cycle, it switches between the cold and warm white leds twice and then stays on the warm white.

**Comment by gadgetmies** [comment]
I left "permit join" on after trying to add the device into my network. Seems that at some point the interview succeeded and now the device is paired and I am able to control it! I do not think I changed anything in the setup and the device has been quite far from the gateway device, but for some reason, what did not work previously, now works. I can now confirm that the device works correctly, but am a bit wary of adding it to the list of supported devices. Perhaps if someone else has this device, can confirm it can be added to zigbee2mqtt and has less issues with adding it to the network, it would be good to add it to the list.

As mentioned earlier, when trying to factory reset the device, I got no feedback from the device after doing the "5x 5s on + 5s off" procedure. This might have been why I was unable to pair the device, but why the pairing worked at some point afterwards, is beyond me. This is also something that would be good to confirm i.e. whether the procedure should work and what kind of feedback the device should give.
